frontend/src/reviews/EditableReviewComment.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Stack, Typography } from '@mui/material'
+import MarkdownDisplay from 'src/common/MarkdownDisplay'
+import RichTextEditor from 'src/common/RichTextEditor'
+import MessageAlert from 'src/MessageAlert'
+import { ResponseInterface } from 'types/types'
+import { formatDateTimeString } from 'utils/dateUtils'
+
+interface EditableReviewCommentProps {
+  comment: string
+  setComment: (value: string) => void
+  response: ResponseInterface
+  isEditMode: boolean
+  setIsEditMode: (value: boolean) => void
```

Comment:
If an error is thrown when updating a comment, it might be useful to reset the error when "Cancel" is clicked. Consider changing this to an `onCancel` prop and in both `ReviewCommentDisplay` and `ReviewDecisionDisplay` we could both `setIsEditMode(false)` and `setEditCommentErrorMessage('')`. That way the error won't be present when they re-enter edit mode.